### ISSUE TITLE
3.1.1 release seems broken

### ISSUE THREAD

**Issue by esimons** [saucelabs/node-saucelabs#78]
Was the publish of version 3.1.1 a mistake? 

I don't see that release in the master branch commit history.
Also, there's no 3.1.0 release on npm.
Also also, there's a 3.1.1 alpha release that is more recent than the actual 3.1.1 release.
(Ref: https://www.npmjs.com/package/saucelabs?activeTab=versions)

Moreover, projects that pull it in are seeing `Error: connect ECONNREFUSED 127.0.0.1:3000` when their test-runner tries to update job details. Rather than hitting the Saucelabs API, it's trying to hit:
`http://localhost:3000/v1/****/jobs/[jobId]`

Thanks!

**Comment by christian-bromann** [comment]
@esimons please update to v4, this release is indeed broken and I will deprecate it.

**Comment by christian-bromann** [comment]
Deprecated test releases. Thanks for pointing this out!

**Comment by alfonso-presa** [comment]
@christian-bromann I'm experiencing this also in my CI (jenkins) environment using wdio v5 sauce service depending on saucelabs@4.0.2 while performing a browser.sessionReload()

```
08:12:43  [4-0] http://eu-central-1.localhost:3000/v1/****/jobs/56b7a8e582604ac1b885929edef0e59d
08:12:43  [4-0] 2020-04-02T06:12:43.023Z ERROR @wdio/sync: Error: getaddrinfo ENOTFOUND eu-central-1.localhost eu-central-1.localhost:3000
08:12:43      at errnoException (dns.js:50:10)
08:12:43      at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)
```

Any guidance?

**Comment by christian-bromann** [comment]
@alfonso-presa can you provide a reproducible example?

**Comment by alfonso-presa** [comment]
@christian-bromann I'm trying to reproduce it locally with out luck.... Could you give me any clue on when that request happens and under which circumstances? I know that it ofcourse happens when you run agains saucelabs with the sauce service included. I've tried to pass the sauce:options capability with all filled in like I do in CI.... No luck yet, and I've confirmed that versions of both sauce service (5.22.5) and saucelabs library are the same (4.0.2)

**Comment by christian-bromann** [comment]
Try to clean your `package-lock.json` and `node_modules` and reinstall everything.

**Comment by alfonso-presa** [comment]
Yep, already tried that :-(

**Comment by christian-bromann** [comment]
Can you check the `saucelabs` code in the node modules folder to see if you find `localhost:3000` in there?

**Comment by alfonso-presa** [comment]
Ok. My bad.... We are using lerna and looks like the different versions of node of my machine and the CI were acting differently with the hoisted dependencies, and we had a package with fixed dependency to `@wdio/sauce-service@5.16.10`

Thank you for the help and sorry for bothering
